spec/ruby/language/range_spec.rb

```diff
@@ -19,6 +19,15 @@
     it "is frozen" do
       (42..).should.frozen?
     end
+
+    it "is not frozen if it is a subclass of Range" do
```

Comment:
This spec should be in a different top-level `describe`, because here it's under `Literal Ranges` but it's not a literal